realm/functions/get_yesterday_youtube_video_stats/source.js

```diff
@@ -0,0 +1,18 @@
+exports = async function () {
+
+  let date = new Date();
+  // Set the date to three days prior (stats will only be returned when YouTube has stats for all metrics)
+  date.setDate(date.getDate() - 3);
+  const year = date.getFullYear();
+  // JavaScript sets January to month 0
+  const month = date.getMonth() + 1;
+  const day = date.getDate();
+  
+  try {
+    await context.functions.execute("get_youtube_video_stats", year, month, day);
+    return `YouTube video stats imported for ${year}-${month}-${day}`;
```

Comment:
I find this a little bit weird - returning a human-readable message from a machine-callable function. Would it be better to return some sort of object describing the date that was imported? It could even include this message in case the client code wants to extract a human-readable message from the returned object.

Reply:
Great point!